WinUIGallery/MainWindow.xaml.cs

```diff
@@ -1,165 +1,116 @@
-
-    public Microsoft.UI.Xaml.Controls.NavigationView NavigationView
+    public NavigationView NavigationView
     {
         get { return NavigationViewControl; }
     }
 
     public Action NavigationViewLoaded { get; set; }
 
-    public DeviceType DeviceFamily { get; set; }
-
-    public string AppTitleText
-    {
-        get
-        {
-#if DEBUG
-            return "WinUI 3 Gallery Dev";
-#else
-            return "WinUI 3 Gallery";
-#endif
-        }
-    }
-
-    public NavigationRootPage()
+    public MainWindow()
     {
         this.InitializeComponent();
         dispatcherQueue = Microsoft.UI.Dispatching.DispatcherQueue.GetForCurrentThread();
 
         _navHelper = new RootFrameNavigationHelper(rootFrame, NavigationViewControl);
-
-        SetDeviceFamily();
-        AddNavigationMenuItems();
-
-        this.GotFocus += (object sender, RoutedEventArgs e) =>
-        {
-            // helpful for debugging focus problems w/ keyboard & gamepad
-            if (FocusManager.GetFocusedElement() is FrameworkElement focus)
-            {
-                Debug.WriteLine("got focus: " + focus.Name + " (" + focus.GetType().ToString() + ")");
-            }
-        };
-
-        // remove the solid-colored backgrounds behind the caption controls and system back button if we are in left mode
-        // This is done when the app is loaded since before that the actual theme that is used is not "determined" yet
-        Loaded += delegate (object sender, RoutedEventArgs e)
-        {
-            NavigationOrientationHelper.UpdateNavigationViewForElement(NavigationOrientationHelper.IsLeftMode(), this);
-
-            Window window = WindowHelper.GetWindowForElement(sender as UIElement);
-            window.Title = AppTitleText;
-            window.ExtendsContentIntoTitleBar = true;
-            window.Activated += Window_Activated;
-            window.SetTitleBar(this.AppTitleBar);
-
-            AppWindow appWindow = WindowHelper.GetAppWindow(window);
-            appWindow.SetIcon("Assets/Tiles/GalleryIcon.ico");
-            appWindow.TitleBar.PreferredHeightOption = TitleBarHeightOption.Tall;
-            _settings = new UISettings();
-            _settings.ColorValuesChanged += _settings_ColorValuesChanged; // cannot use FrameworkElement.ActualThemeChanged event because the triggerTitleBarRepaint workaround no longer works
-        };
     }
 
-    private void Window_Activated(object sender, WindowActivatedEventArgs args)
+    private void RootGrid_Loaded(object sender, RoutedEventArgs e)
     {
-        if (args.WindowActivationState == WindowActivationState.Deactivated)
-        {
-            VisualStateManager.GoToState(this, "Deactivated", true);
-        }
-        else
-        {
-            VisualStateManager.GoToState(this, "Activated", true);
-        }
+        NavigationOrientationHelper.UpdateNavigationViewForElement(NavigationOrientationHelper.IsLeftMode());
+#if DEBUG
+        this.Title = "WinUI 3 Gallery Dev";
```

Comment:
I'm thinking - the Title can be fixed now, if "Dev" is the subtitle?

Reply:
This is for the title of the window, not the titlebar. We are using the Subtitle for titlebar (see line below).